lib/engine/game/g_1846.rb

```diff
@@ -314,6 +314,23 @@ def event_close_companies!
         super
 
         @minors.dup.each { |minor| close_corporation(minor) }
+
+        %w[D14 E17].each do |hex|
+          hex_by_id(hex).tile.icons.reject! { |icon| icon.name == 'lsl' }
+        end
+      end
+
+      def event_remove_private_tokens!
+        # TODO trigger this
+        %w[B8 C5 D14 G19 I1].each do |hex|
+          hex_by_id(hex).tile.icons.reject! { |icon| icon.name == 'port' }
+        end
+
+        %w[D6 I1].each do |hex|
```

Comment:
are there any icons in the game? imo just simplify this, make this one array and just cleare all icons?

Reply:
oh yeah that would prob work better